### PR TITLE
Upgrade button-group to v2.11.0

### PR DESCRIPTION
**Pull request by tomnez**
Diff [here](https://github.com/Shopify/polaris/compare/v2.2.0...v2.11.0#diff-ae076cc3c15cddac2f460c6941d4626f)

Finishes https://github.com/smile-io/ember-polaris/issues/229

### CODE REVIEW

File: tests/integration/components/polaris-button-group-test.js [hunk 1]
```diff
@@ -57,6 +54,40 @@ test('it renders the correct HTML with segmented attribute', function(assert) {
   );
 });
 
+test('it renders the correct HTML with fullWidth attribute', function(assert) {
+  this.set('fullWidth', true);
+  this.render(hbs`{{polaris-button-group fullWidth=fullWidth}}`);
+
+  const buttonGroup = find(buttonGroupSelector);
+  assert.ok(
```
Comment (vladucu) [inline tests/integration/components/polaris-button-group-test.js]: why not something simpler
`assert.dom(buttonGroupSelector).hasClass('Polaris-ButtonGroup--fullWidth', 'fullWidth=false - does not add the fullWidth class');`

Comment (tomnez) [reply tests/integration/components/polaris-button-group-test.js]: Doh! Forgot to codemod this file..

**Comment by tomnez** [comment]
@vladucu updated

**Comment by tomnez** [comment]
lol @vladucu updated again.

**Pull request merged**